src/test/java/org/opensearch/flowframework/FlowFrameworkRestTestCase.java

```diff
@@ -292,17 +292,27 @@ protected void wipeAllODFEIndices() throws IOException {
 
             for (Map<String, Object> index : parserList) {
                 String indexName = (String) index.get("index");
-                // Do not reset ML/Flow Framework encryption index as this is needed to encrypt connector credentials
-                if (indexName != null
-                    && !".opendistro_security".equals(indexName)
-                    && !".plugins-ml-config".equals(indexName)
-                    && !".plugins-flow-framework-config".equals(indexName)) {
-                    adminClient().performRequest(new Request("DELETE", "/" + indexName));
+                if (indexName != null) {
+                    // Filter out system index deletion only when running security enabled tests
+                    if (isHttps() && isSystemIndex(indexName)) {
+                        continue;
+                    }
+                    // Do not reset ML/Flow Framework encryption index as this is needed to encrypt connector credentials
+                    if (!".plugins-ml-config".equals(indexName) && !".plugins-flow-framework-config".equals(indexName)) {
```

Comment:
we'll never get here since these are both system indices, right?

Reply:
In a non-security enabled cluster this will ensure the encryption indices are still filtered out